metrics.md

```diff
@@ -0,0 +1,226 @@
+# Side-Car Metrics
+
+This document describes the various instrumentation points that are available in the side-car. This should be utilized to monitor the health of the side-car and the services it is proxying.
+
+If there are any additional metrics that you would like to see, please open an issue in the [GitHub repository](https://github.com/skip-mev/slinky). Note that this document is not fully comprehensive and may be updated in the future. However, it should provide a good starting point for monitoring the side-car.
+
+# Table of Contents
+
+* [Metrics](#metrics)
+    * [Health Metrics](#health-metrics)
+    * [Prices Metrics](#prices-metrics)
+        * [Price Feed Metrics](#price-feed-metrics)
+        * [Aggregated Price Metrics](#aggregated-price-metrics)
+    * [HTTP Metrics](#http-metrics)
+    * [WebSocket Metrics](#websocket-metrics)
+
+# Metrics
+
+> **Definitions**:
+>
+> * **Market**: A market is a pair of assets that are traded against each other. For example, the BTC-USD market is the market where Bitcoin is traded against the US Dollar.
+> * **Price Feed**: A price feed is indexed by a price provider and a market. For example, the Coinbase API provides a price feed for the BTC-USD market.
+> * **Price Provider**: A price provider is a service that provides price data for a given market. For example, the Coinbase API is a price provider for the Coinbase markets.
+> * **Market Map Provider**: A market map provider is a service that supplies the markets that the side-car needs to fetch data for.
+
+The side-car exposes metrics on the `/metrics` endpoint. These metrics are in the Prometheus format and can be scraped by Prometheus or any other monitoring system that supports Prometheus format.
```

Comment:
is there a default port this is exposed on?

Reply:
this is the default that prom sets up whenever you run a prom server

Reply:
but i think the port we expose should be explicitly defined, ill update